### PR TITLE
Added support for wellplated.com

### PR DESCRIPTION
**Pull request by jknndy**

### CODE REVIEW

File: recipe_scrapers/wellplated.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def instructions(self):
+        instructions = self.schema.instructions()
+        if isinstance(instructions, list):
+            instructions = "\n".join(instructions)
```
Comment (jayaddison) [inline recipe_scrapers/wellplated.py:32]: Out of curiosity: did you find pages on this website where the `instructions` was not a list?

Comment (jknndy) [reply recipe_scrapers/wellplated.py]: After looking back through there doesn't appear to be any. I likely carried that over from another scraper by mistake. I've removed lines 31 & 32

**Comment by jknndy** [comment]
I'm not sure there is anything I can do to resolve whatever is causing these fails. It appears to be related to the test python instance's SSL connection according to the logs.

**Comment by jayaddison** [comment]
> I'm not sure there is anything I can do to resolve whatever is causing these fails. It appears to be related to the test python instance's SSL connection according to the logs.

Whatever the cause, it seems to have been a temporary/intermittent failure - I hadn't seen those errors occur before.  Re-running the failed GitHub Actions jobs has allowed the checks here to pass.

**Review by jayaddison — APPROVED** [review]
Looks good; thanks @jknndy.  I'll likely do another round of code review and merging within the next 8 hours or so.

File: recipe_scrapers/wellplated.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def instructions(self):
+        instructions = self.schema.instructions()
+        return instructions
```
Comment (jayaddison) [inline recipe_scrapers/wellplated.py:31]:
```suggestion
        return self.schema.instructions()
```

**Comment by jayaddison** [comment]
This has a similar condition to #801: there's author information available on the page, but it's not currently available to the `schema.org` extraction code we have in place.

Perhaps it's worth considering those as improvements instead of blockers for merge.  Ideas/feedback welcome.

**Comment by jknndy** [comment]
Took the same approach here as with the other one. Implemented a html scraper to grab the author from another section of the webpage.

**Comment by jayaddison** [comment]
Nice - looks good to me; thanks again @jknndy!

**Pull request merged**